aqua/qt5/Portfile

```diff
@@ -1532,6 +1533,19 @@ foreach {module module_info} [array get modules] {
 
             # special case
             if { ${module} eq "qtwebkit" } {
+                # in Xcode 13, for some reason the include search is not case sensitive
+                # in some cases, this causes the compiler to find the wrong header in
+                # macports' default install directory example: archive.h vs Archive.h
+                # See https://trac.macports.org/ticket/63877
```

Comment:
I don't understand. When has include search ever been case sensitive except on a case sensitive filesystem?

Reply:
Happy to change the comment to whatever you think is the correct wording.  

The real issue was that the include search was finding the wrong file since a similarly named header file was in macports install directory.  Removing the initial "-I${prefix}/include" flag fixed the issue - and since it gets added back onto the path later did not hurt the compile.

```
loader/FrameLoader.cpp:130:10: warning: non-portable path to file '"archive.h"'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include "Archive.h"
         ^~~~~~~~~~~
         "archive.h"
```

Reply:
Perhaps "in Xcode 13 the include search path no longer looks locally for headers first, so if any header files with conflicting names exist in ${prefix}/include the compiler will fail"

Reply:
alternatively "Fix issue in Monterey where the include search path finds and tries to link against similarly named header files in "${prefix}/include" that conflict with webkit's own header files."

Reply:
@ryandesign I just pushed the revision to the comment - apologies for the confusion on the actual problem.  Please let me know if this resolves the concern or if I should make additional changes.